squirrel.worker/src/test/resources/rdf_analyzer/new_york/RDFXML

```diff
@@ -0,0 +1,7981 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<rdf:RDF
+	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+	xmlns:owl="http://www.w3.org/2002/07/owl#"
```

Comment:
As explained above, having a file with nearly 8000 lines of codes does not make much sense for a test case (unless we would like to test some kind of scalability). I would suggest to use a much smaller XML file that has a small number of triples but the triples are representing different kinds of triples.

Reply:
I have created a sample RDFXML file with less triples enough to analyze

Reply:
I have removed the other RDFXML file created before.